### ISSUE TITLE
JSON logging sometimes emits invalid JSON

### ISSUE THREAD

**Issue by tjennings** [javalite/activeweb#334]
For example: 

Example:

{"level":"WARN","timestamp":"Fri Mar 31 16:42:05 CDT 2017","thread":"pool-2-thread-1","logger":"com.jes.services.SynchronizedScheduler","message":"Skip task "AsyncReprocessor". Task was executed at Fri Mar 31 16:37:05 CDT 2017 on webapp2.stage"}

{"level":"INFO","timestamp":"Fri Mar 31 16:37:07 CDT 2017","thread":"catalina-exec-5","logger":"org.javalite.activeweb.controller_filters.RequestParamsLogFilter","message":{}, session: null}

**Comment by ipolevoy** [comment]
this seems to be related to how a message is logged at the application level. Looking at this log, I can imagine this code: 

     logger.warn("Skip task \"" + name +"\". Task was executed at " + new Date(..) + " on webapp2.stage");

Since you configured JSON logging for your app,  you need to pass correct content so as not to break JSON parsing. I suggest you do it: 

    logger.warn("Skip task '" + name +"'. Task was executed at " + new Date(..) + " on webapp2.stage");

In other words, do not pass things into a log that may break JSON parsing
